### PR TITLE
[Documentation] Fixing typos in documentation

### PR DESCRIPTION
**Pull request by xbtmatt**
### Description
Adding factory methods for MoveVector for AccountAddress and MoveOption:

```typescript
MoveVector.AccountAddress
MoveOption.AccountAddress
```

This is from an old PR

### Test Plan
added unit tests
`pnpm jest`

### CODE REVIEW

File: src/bcs/serializable/moveStructs.ts [hunk 1]
```diff
@@ -432,6 +447,32 @@ export class MoveOption<T extends Serializable & EntryFunctionArgument>
...
+   */
+  static AccountAddress(value?: HexInput | AccountAddress | null): MoveOption<AccountAddress> {
+    let accountAddress: AccountAddress | undefined;
+    if (value !== null && value !== undefined) {
+      if (value instanceof AccountAddress) {
+        accountAddress = value;
+      } else {
+        accountAddress = AccountAddress.fromHexInputRelaxed(value);
+      }
+    } else {
```
Comment (gregnazario) [inline src/bcs/serializable/moveStructs.ts:470]: We should merge this in with this change https://github.com/aptos-labs/aptos-ts-sdk/pull/145 but it keeps bloating more and more for me every time I'm fixing one thing, so I'm trying to get it working soon.

File: src/bcs/serializable/moveStructs.ts [hunk 2]
```diff
@@ -184,13 +180,31 @@ export class MoveVector<T extends Serializable & EntryFunctionArgument>
...
+   * @params values: an array of `HexInput | AccountAddress`
+   * @returns a `MoveVector<AccountAddress>`
+   */
+  static AccountAddress(values: Array<HexInput | AccountAddress>): MoveVector<AccountAddress> {
+    const accountAddresses = values.map((v) => {
+      if (v instanceof AccountAddress) {
+        return v;
+      }
+      return AccountAddress.fromHexInputRelaxed(v);
+    });
```
Comment (gregnazario) [inline src/bcs/serializable/moveStructs.ts:204]: This seems like it could be put in a helper function.

**Review by gregnazario — APPROVED** [review]
Not really opposed to adding these extra factory functions, but it does really just help you if you're doing manual BCS encoding right?

**Comment by xbtmatt** [comment]
@gregnazario Yea, tbh I was just adding because it was ready to be merged- but ultimately I haven't had a need for these functions so I don't think we need them.

This PR can just be an update for the docs, I don't want to make things needlessly more bloated/complex, so I removed the factory methods

**Pull request merged**